salt/modules/rpm.py

```diff
@@ -478,16 +478,22 @@ def info(*packages):
                 continue
             key, value = line
             key = key.replace(' ', '_').lower()
+            if key != 'name' and filter_attrs and key not in filter_attrs:
```

Comment:
No. This will introduce a regression by bypassing full list of attributes by empty `attr`.